start.sh

```diff
@@ -76,8 +76,6 @@ prepare_configs
 [ -n "${INSTALL_WEB_SERVER}" ] && change_setting "INSTALL_WEB_SERVER" "$INSTALL_WEB_SERVER"
 [ -n "${INSTALL_WEB_INTERFACE}" ] && change_setting "INSTALL_WEB_INTERFACE" "$INSTALL_WEB_INTERFACE"
 [ -n "${LIGHTTPD_ENABLED}" ] && change_setting "LIGHTTPD_ENABLED" "$LIGHTTPD_ENABLED"
-[ -n "${ServerIP}" ] && changeFTLsetting "REPLY_ADDR4" "$ServerIP"
```

Comment:
For ease of updating, I think we should leave these two lines in - but add a note to the README/release notes that they will be removed in a later version, and should be replaced with `FTLCONF_REPLY_ADDR4/6` 

I'll look at untangling `ServerIP` from the rest of the codebase at some point:
![image](https://user-images.githubusercontent.com/1998970/143308548-925c123f-48b1-492d-b82f-fedfc6cc2cf4.png)